### PR TITLE
Add support to detect supported features in grabbercontrols2gui 

### LINKED ISSUE #149: Support to detect supported features in grabbercontrols2gui
I designed `grabbercontrols2gui` to be specific for our GigE camera and its current needs, and hence its old name, `AravisGigEController` and the supported subset of camera features.

Since we want to use this GUI with many devices, some of those specific features might not be present in every grabber, so it would be great to detect which of those features are not available (I'm thinking of zoom, for instance) and disable the corresponding controls.

### PR DESCRIPTION
**Pull request by David-Estevez**
Fixes #149

### CODE REVIEW

File: programs/GrabberControls2Gui/GrabberControls2GuiGUI.py [hunk 1]
```diff
@@ -102,6 +102,18 @@ def setupUI(self):
         self.fpsSpinBox.setMinimum(0)
         self.fpsSpinBox.setMaximum(30)
 
+        # Disable widgets if feature not supported
+        self.zoomSlider.setEnabled(False) if not self.controller.has_zoom() else None  # Look me mom I'm a hacker!
```
Comment (PeterBowman) [inline programs/GrabberControls2Gui/GrabberControls2GuiGUI.py]: Tip for pro hackers 😄:

```python
not self.controller.has_zoom() and self.zoomSlider.setEnabled(False)
```

Although this feels far less obscure:

```python
self.zoomSlider.setEnabled(self.controller.has_zoom())
```

Or just group in pairs like an old-sk00ler, provided `True` is the default value (it seems so):

```python
if not self.controller.has_zoom():
    self.zoomSlider.setEnabled(False)
    self.zoomSpinBox.setEnabled(False)
```

File: libraries/YarpPlugins/AravisGigE/CMakeLists.txt [hunk 2]
```diff
@@ -18,8 +18,7 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR} # yarp plugin builder needs this
                     ${_common_dirs}
                     ${ARAVIS_INCLUDE_DIRS})
 
-yarp_add_plugin(AravisGigE AravisGigE.cpp
```
Comment (PeterBowman) [inline libraries/YarpPlugins/AravisGigE/CMakeLists.txt:21]: [My bad](https://github.com/roboticslab-uc3m/yarp-devices/commit/b297cde1061b0d3210809abb6e23c93d077d42c9#diff-9ecb81e93a9d6bd6d241586d3183cc91R21), sorry!

**Comment by jgvictores** [comment]
@PeterBowman ...what is the state of this?

**Comment by PeterBowman** [comment]
Not tested, LGTM though.

**Comment by jgvictores** [comment]
@David-Estevez Good enough for you to merge?

**Comment by David-Estevez** [comment]
I want to implement what @PeterBowman suggested, then it'll be ok to merge.

**Comment by jgvictores** [comment]
Ok!

**Comment by David-Estevez** [comment]
Rebased and ready to merge :+1:

**Pull request merged**